### PR TITLE
flex section js and css added

### PR DESCRIPTION
**Pull request by JDRoy93**
https://feature-flex-section--metafox-sites--bmw-importer.hlx.page/test-pages/index-copy/page-with-deafultblocks

### CODE REVIEW

File: blocks/flexible-width-section/flexible-width-section.css [hunk 1]
```diff
@@ -28,21 +28,47 @@
...
 }
 
-.section .right {
+.section.right.grid-6,
+.section.right.grid-8,
+.section.right.grid-10  {
   text-align: right;
+}
+
+.section.center.grid-6 .button-container > .align-icon,
```
Comment (Buuhuu) [inline blocks/flexible-width-section/flexible-width-section.css:49]: You can consolidate these listing of long selectors with `:where()`

```.section.center:where(.grid-6,.grid-8,.grid-10) .button-container > .align-icon```

Comment (JDRoy93) [reply blocks/flexible-width-section/flexible-width-section.css]: Updated

**Review by Harish-DV — CHANGES_REQUESTED** [review]
Comments from Adobe

why is identifier hidden? without a condition it will never show, no?
if you insist on having multiple properties on the section instead of a multi-select style field, better give them meaningful names like gridWidth and alignment instead of styles and classes. Only the styles key has a special purpose and becomes a css class. Everything else is just a data attribute. You can use the data attributes in css selectors as well so that is fine.
you do not need a custom decorateSections implementation. Section metadata becomes data attributes which are accessible in css selectors. If you want them to be class names you can use your own decorate function in addition to the one from aem.js (as in runs after decorateSections, reads data attributes and converts them to classes)
you must not remove decorateSections from decorateMain
if you add custom decorations to decorateMain make sure to add them to the editor-support.js script as well

**Review by bharath-yathiraj — APPROVED** [review]
approved

**Review by Harish-DV — APPROVED** [review]

**Pull request merged**